ext/sftp/client.go

```diff
@@ -17,7 +17,10 @@ func newClient(address, username, password, privateKey, hostFingerprint string)
 		User:            username,
 		Auth:            []ssh.AuthMethod{},
 		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
-		Timeout:         5 * time.Second,
+		Timeout:         5 * time.Minute,
+		HostKeyAlgorithms: []string{
+			ssh.KeyAlgoRSA,
```

Comment:
`RSA` should not be a default configuration here, as by default, newest `ssh` is not supported `RSA` algo as a default. can we remove it? or at least if it's required, better to expose it as a configuration

Reply:
Right now the google's SFTP servers are only supporting these three algorithms. However, since the default ssh config is already covering this, default value should work. But if in the future ssh default value doesn't contain these values, then we'll need to manually configure this

```go
[]string{ssh.KeyAlgoRSA, ssh.KeyAlgoRSASHA256, ssh.KeyAlgoRSASHA512}
```

Reply:
ack, let's remove it for now. later on we can add this as configurable option

Reply:
Let's remove this for now